### DISCUSSION TITLE
Conexão Flutter e SolidWorks

### DISCUSSION THREAD

**Discussion by Mathsfer07** [Flutterando/forum#135] [Discussões gerais]
Olá, alguém sabe se é possível fazer uma aplicação desktop em Flutter para pegar informações do SolidWorks?

**Comment by ghost** [comment]
> Olá, alguém sabe se é possível fazer uma aplicação desktop em Flutter para pegar informações do SolidWorks?

Você se refere a página inicial, todas as páginas ou alguma página em específico? 

Geralmente para fazer alguma dessas 3 opções, o mais aconselhável seria ao meu ver utilizar o Python com o Selenium, já que o Selenium é um framework para você consultar determinadas páginas. E o python por ser uma linguagem simples, você consegue fazer essas consultas.

Como sou novo no flutter não sei se existe suporte do Selenium no flutter de forma oficial e não tercerizada, quem souber de alguma informação extra comente abaixo.

Espero ajudar, caso não ajude, vou ver como ajudar de outra forma.
